packages/js-example-app/src/index.client.ts

```diff
@@ -0,0 +1,214 @@
+import {
+  BlrButtonIcon,
+  BlrButtonText,
+  BlrCheckbox,
+  BlrInputFieldNumber,
+  BlrInputFieldText,
+  BlrRadio,
+  BlrSelect,
+  BlrTabBar,
+  BlrTextarea,
+  BlrToggleSwitch,
+} from '@boiler/ui-library';
+
+await hydrate();
```

Comment:
Functionality might be ok but simply for the properness of the code flow, declare the functions first and then function call later on. Same for line 15.